dashboard/config/locales/en.yml

```diff
@@ -1021,6 +1021,81 @@ en:
       things_you_liked_s: 'What were the two things you liked most about the activities you did in this workshop and why?'
       things_you_would_change_s: 'What are the two things you would change about the activities you did in this workshop?'
       anything_else_s: 'Is there anything else you’d like to tell us about your experience at this workshop?'
+    form_labels:
```

Comment:
I think @tanyaparker may have suggested putting this in a new file for forms i18n?

Reply:
@Hamms What do you think?  Is it worth making a new `.yml` file for this form?

Reply:
Hmmm, that's a good question. I certainly wouldn't want to create a new i18n file just for this form, but I could definitely see making a new i18n file for form content in general. Do you have a sense of how much content would end up in that file and/or how many new forms of this type we intend to make in the future?

Reply:
I'm not aware of any more loc'ed forms that are currently planned to be implemented in this system.  We could certainly extract these strings if/when that time comes, right?